notebook/tests/selenium/test_deletecell.py

```diff
@@ -1,10 +1,5 @@
 import os
 import pytest
-from .utils import Notebook
-
-@pytest.fixture
-def notebook(authenticated_browser):
-    return Notebook.new_notebook(authenticated_browser)
 
 def get_cells_contents(nb):
```

Comment:
@takluyver  Thanks for taking the time to review my PR. I made some changes to address your comments. I have only one more idea that I'm not sure about and I would like your input. I was thinking of moving this function to the `Notebook` class since I think it might be needed/repeated in several tests. I'm not sure about other helper functions like `delete_cell`, `cell_is_deletable` and `set_cell_metadata`, do you think they also should be moved to the `Notebook` class too?

Reply:
I'd say that the `get_cells_contents` and `set_cell_metadata` can move to the notebook class, but let's leave `delete_cell` and `cell_is_deletable` here for now. They can always be moved in another PR if we change our minds. :-)

Reply:
I made the changes. I believe the PR is ready to be merged if you don't have any more comments :)